ethtool: add examples

aherst opened the pull request:
Add examples to print current settings, driver information, usage statistics, and to blink the LEDs.

<!-- Thank you for sending a PR! -->
<!-- Please perform the following checks and mark all the boxes accordingly. -->
<!-- You can remove the checklist items that don't apply to your PR. -->

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

bl-ue commented:
@ahest will you please change the short options to long options in all of the examples (including the ones you didn't add), while we're here?

aherst commented:
@bl-ue I've changed the examples I've added. There doesn't seem to be a long option for -s. 

Is the tldr style to use long options wherever possible?

bl-ue commented:
Yep 👍🏻

bl-ue commented on pages/linux/ethtool.md:
```diff
@@ -3,6 +3,22 @@
+
+`ethtool --statistics {{eth0}}`
+
+- Blink one or more LEDs on an interface for 10 seconds:
```
I always this this it's cool that you can do this kind of stuff via the CLI (software, in general) 🚀 😄

aherst replied:
Unfortunately, it's working on fewer and fewer network cards. But when you do need it and it does work, it's a real time saver!

navarroaxel commented on pages/linux/ethtool.md:
```diff
@@ -3,6 +3,22 @@
 > Display and modify Network Interface Controller (NIC) parameters.
 > More information: <http://man7.org/linux/man-pages/man8/ethtool.8.html>.
 
+- Print the current settings for an interface:
```
I prefer the verb `display`:
```suggestion
- Display the current settings for an interface:
```

aherst replied:
Changed as requested.

navarroaxel commented on pages/linux/ethtool.md:
```diff
@@ -3,6 +3,22 @@
+
+`ethtool {{eth0}}`
+
+- Print the driver information for an interface:
```
```suggestion
- Display the driver information for an interface:
```

aherst replied:
Changed as requested.

navarroaxel commented on pages/linux/ethtool.md:
```diff
@@ -3,6 +3,22 @@
+
+`ethtool --driver {{eth0}}`
+
+- Print the network usage statistics for an interface:
```
```suggestion
- Display the network usage statistics for an interface:
```

aherst replied:
Changed as requested.

owenvoke reviewed (approved):
LGTM after the minor changes from @navarroaxel.
